draft-ietf-gnap-core-protocol.md

```diff
@@ -3326,6 +3326,60 @@ NOTE: '\' line wrapping per RFC 8792
 
 \[\[ [See issue #103](https://github.com/ietf-wg-gnap/gnap-core-protocol/issues/103) \]\]
 
+### Binding a New Key to the Rotated Access Token {#rotate-access-token-key}
+
+If the client instance wishes to bind a new presentation key to an access token, the client
+instance MUST present both the new key and previous key in the access token rotation request.
+The client instance makes an HTTP POST as a JSON object with the following fields:
```

Comment:
Nit: maybe `an HTTP POST request` for consistency reasons? The specification uses two forms `an HTTP POST request` and `an HTTP POST` at the same time.